tests/v2/validation/prime/prime_test.go

```diff
@@ -0,0 +1,81 @@
+package prime
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+const rancherBrand = "suse"
+
+type PrimeVersionTestSuite struct {
+	suite.Suite
+	session        *session.Session
+	client         *rancher.Client
+	isPrime        bool
+	rancherVersion string
+}
+
+func (t *PrimeVersionTestSuite) TearDownSuite() {
+	t.session.Cleanup()
+}
+
+func (t *PrimeVersionTestSuite) SetupSuite() {
+	testSession := session.NewSession(t.T())
+	t.session = testSession
+	primeConfig := new(Config)
```

Comment:
nit - maybe we can break this up so it's not as cluttered? So a space after line 33 and a space after line 35?